rest/src/test/java/at/jku/isse/ecco/rest/RestTest.java

```diff
@@ -0,0 +1,35 @@
+package at.jku.isse.ecco.rest;
+
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.runtime.EmbeddedApplication;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+//Do not run with Grade use Intellij IDEE or similar instead
```

Comment:
this would mean that the build scripts are missing something. If IntelliJ is needed, it's harder to run tests in a pipeline. Please fix so they can run with gradle.

Reply:
just tested it after switching from testng to junit, now it works with gradle as well